docs/reference/crds/verifiers.md

```diff
@@ -45,13 +47,54 @@ spec:
 | trustPolicyDoc   | yes     |   [Trust policy](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md) is a policy language that indicates which identities are trusted to produce artifacts.          |     ""    |
 
 ## Cosign
-Doc coming soon..
+Cosign verifier can be used to verify signatures generated using [cosign](https://github.com/sigstore/cosign/), learn more about the plugin [here](../../../plugins/verifier/cosign/README.md)
+```yml
+apiVersion: config.ratify.deislabs.io/v1alpha1
+kind: Verifier
+metadata:
+  name: verifier-cosign
+spec:
+  name: cosign
+  artifactTypes: application/vnd.dev.cosign.artifact.sig.v1+json
+  parameters:
+    key: /usr/local/ratify-certs/cosign/cosign.pub
+```
+| Name        | Required | Description | Default Value |
+| ----------- | -------- | ----------- | ------------- | 
+| key      | yes    |     Path to the public key used for validating the signature    |   ""            |
 
 ## Sbom
-Doc coming soon..
+```yml
+apiVersion: config.ratify.deislabs.io/v1alpha1
```

Comment:
want to point out that @binbin-li has a PR that will update v1alpha1 --> v1beta1. I think we would need to update this too depending on the order it gets merged.

Reply:
Discussed during community call: this PR will go before @binbin-li PR.